hooks/packer_fmt.sh

```diff
@@ -17,14 +17,21 @@ source "$SCRIPT_DIR/../lib/util.sh"
 
 util::parse_cmdline "$@"
 
-error=0
-
+pids=()
 for file in "${FILES[@]}"; do
-  if ! packer fmt "${ARGS[@]}" -- "$file"; then
+  # Check each path in parallel
+  {
+    packer fmt "${ARGS[@]}" -- "$file"
+  } &
```

Comment:
Why are the braces necessary here?  Is it just to maintain consistency with the `packer_validate.sh` code?

Reply:
Precisely. I wanted to keep the form consistent so if anything was added to the `packer_fmt` hook it would just get dumped into an existing command block.